src/nft/components/bag/Bag.tsx

```diff
@@ -322,57 +287,52 @@ const Bag = () => {
       {bagExpanded && shouldShowBag ? (
         <Portal>
           <Column zIndex={isMobile || isOpen ? 'modal' : '3'} className={styles.bagContainer}>
-            <BagHeader numberOfAssets={itemsInBag.length} toggleBag={toggleBag} resetFlow={reset} />
-            {itemsInBag.length === 0 && bagStatus === BagStatus.ADDING_TO_BAG && <EmptyState />}
-            <ScrollingIndicator top show={userCanScroll && scrollProgress > 0} />
-            <Column ref={scrollRef} className={styles.assetsContainer} onScroll={scrollHandler} gap="12">
-              <Column display={priceChangedAssets.length > 0 || unavailableAssets.length > 0 ? 'flex' : 'none'}>
-                {unavailableAssets.length > 0 && (
-                  <UnavailableAssetsHeaderRow
-                    assets={unavailableAssets}
-                    usdPrice={fetchedPriceData}
-                    clearUnavailableAssets={() => setItemsInBag(availableItems)}
-                    didOpenUnavailableAssets={didOpenUnavailableAssets}
-                    setDidOpenUnavailableAssets={setDidOpenUnavailableAssets}
-                    isMobile={isMobile}
+            {!(isProfilePage && profilePageState === ProfilePageStateType.LISTING) ? (
```

Comment:
maybe move this to a variable named `isOnProfileListingPage` just so logically its easier to decipher when being read

Reply:
I personally prefer this setup as this var would only be use in 1 spot and I think this is very readable since there's only the two conditions: "I'm on the profile page and currently listing"